src/components/UserInfo/UserInfo.tsx

```diff
@@ -1,5 +1,16 @@
 import React from 'react';
+import { User } from '../../types/User';
 
-export const UserInfo: React.FC = () => (
-  <>Put the user here</>
+interface Props {
+  user: User;
+}
+
+export const UserInfo: React.FC<Props> = ({ user: { email, name } }) => (
+  <p>
+    {' Posted by  '}
```

Comment:
P.S. but if you need a space before `name` use like this `Posted by{' '}`
```suggestion
    Posted by
```